net/ddns-scripts/Makefile

```diff
@@ -9,10 +9,10 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=ddns-scripts
 # Version == major.minor.patch
 # increase on new functionality (minor) or patches (patch)
-PKG_VERSION:=2.7.6
+PKG_VERSION:=2.7.7
 # Release == build
 # increase on changes of services files or tld_names.dat
-PKG_RELEASE:=21
+PKG_RELEASE:=22
```

Comment:
If you increase PKG_VERSION, whats OK here then you need to RESET (set to 1) the PKG_RELEASE

Reply:
fixed